share/tools/web_config/sample_prompts/nim.fish

```diff
@@ -31,53 +66,41 @@ function fish_prompt
     end
     echo -n (prompt_hostname)
     set_color -o white
-    #echo -n :(prompt_pwd)
-    echo -n :(pwd|sed "s=$HOME=~=")
+    echo -n :(prompt_pwd)
     set_color -o green
     echo -n ']'
-    set_color normal
-    set_color $retc
-    if [ $tty = tty ]
-        echo -n '-'
-    else
-        echo -n '─'
-    end
-    set_color -o green
-    echo -n '['
-    set_color normal
-    set_color $retc
-    echo -n (date +%X)
-    set_color -o green
-    echo -n ]
 
-    if type -q acpi
-        if [ (acpi -a 2> /dev/null | string match -r off) ]
-            echo -n '─['
-            set_color -o red
-            echo -n (acpi -b|cut -d' ' -f 4-)
-            set_color -o green
-            echo -n ']'
-        end
-    end
+    # Date
+    _nim_prompt_wrapper $retc '' (date +%X)
+
+    # Virtual Environment
+    set -q VIRTUAL_ENV
+    and _nim_prompt_wrapper $retc V (basename "$VIRTUAL_ENV")
+
+    # git
+    test -d .git
```

Comment:
`__fish_git_prompt` already figures out if there's a git repo, so this is unnecessary.

Reply:
I need it to know if I have to put the extra `─[]` or not.
In other words, if I remove it, the minimalist prompt would be:
```
┬─[nim@Hattori:~]─[11:39:00]─[]
╰─>$ echo here
```

Reply:
In that case, you'll want to `test -n` on the output of `__fish_git_prompt`.

Even if there is a `.git` directory, that's not a guarantee that there's an actual git repo - just run `mkdir .git` somewhere.

Reply:
right, thanks :)

Reply:
arg, it doesn't work, and gives `─[G:]`, did I missed something ?

Reply:
fixed :)